e2e/pages/app/wizard/wizard.testpage.component.html

```diff
@@ -12,8 +13,9 @@
         <p>Step 3 Content</p>
     </ux-wizard-step>
 
-    <ux-wizard-step header="Step 4">
+    <ux-wizard-step header="Step 4" [disableNextWhenInvalid]="disableNextWhenInvalid" [valid]="!stepInvalid">
         <p>Step 4 Content</p>
+        <ux-checkbox [(value)]="stepInvalid" id= "stepInvalid">Set this step to error state</ux-checkbox>
```

Comment:
I'd recommend not having two components with the same `id` value. I think these two checkboxes can have separate ids and values, e.g. `step1Invalid`, `step4Invalid`.

Also there's a random extra space in here and above which should be removed.